client/deposits/index.tsx

```diff
@@ -118,7 +118,7 @@ const DepositFailureNotice: React.FC = () => {
 		>
 			{ interpolateComponents( {
 				mixedString: __(
-					'Deposits are currently paused because a recent deposit failed. Please {{updateLink}}update your bank account details{{/updateLink}}.',
+					'Payouts are currently paused because a recent attempt failed. Please {{updateLink}}update your bank account details{{/updateLink}}.',
```

Comment:
Would be good to get design/product feedback on this wording, I'm not sure which of `attempt` vs double `deposit` is better!